<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetParticle</class>
 <widget class="QWidget" name="WidgetParticle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>1239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>编辑粒子效果</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>粒子名称</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="name"/>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QCheckBox" name="modelSpace">
       <property name="text">
        <string>使用模型坐标</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="emitterPlane">
       <property name="text">
        <string>跟随平面移动</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QCheckBox" name="lockEmitter">
       <property name="text">
        <string>固定位置</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="xyQuad">
       <property name="text">
        <string>始终向上</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QCheckBox" name="blend">
       <property name="text">
        <string>加法混合</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="squirt">
       <property name="text">
        <string>喷射</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QCheckBox" name="zWrite">
       <property name="text">
        <string>写入深度缓冲</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="faceSpeed">
       <property name="text">
        <string>朝向移动方向</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>绑定骨骼</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="boneId"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>移动速度</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="speed"/>
     </item>
     <item>
      <widget class="QLineEdit" name="speedVar"/>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>随机角度</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="coneAngle"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>粒子重力</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLineEdit" name="gravityX"/>
     </item>
     <item>
      <widget class="QLineEdit" name="gravityY"/>
     </item>
     <item>
      <widget class="QLineEdit" name="gravityZ"/>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>生命周期</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="lifeSpan"/>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QCheckBox" name="enableLifeRandom">
     <property name="text">
      <string>启用随机生命周期</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>随机周期</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLineEdit" name="lifeRandom1"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lifeRandom2"/>
     </item>
    </layout>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>发射数量</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLineEdit" name="emissionRate"/>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_24">
     <property name="text">
      <string>随机偏移</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QLineEdit" name="width"/>
     </item>
     <item>
      <widget class="QLineEdit" name="height"/>
     </item>
    </layout>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="QCheckBox" name="tailMode">
     <property name="text">
      <string>启用尾迹模式</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>尾迹长度</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>尾迹速度</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="0">
      <widget class="QCheckBox" name="timeBasedCell">
       <property name="text">
        <string>启用序列</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="matchLife">
       <property name="text">
        <string>序列对应生命周期</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>贴图序列</string>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="row"/>
     </item>
     <item>
      <widget class="QLineEdit" name="col"/>
     </item>
    </layout>
   </item>
   <item row="16" column="0">
    <widget class="QLabel" name="label_26">
     <property name="text">
      <string>序列帧率</string>
     </property>
    </widget>
   </item>
   <item row="16" column="1">
    <widget class="QLineEdit" name="uvAnimFps"/>
   </item>
   <item row="17" column="0">
    <widget class="QLabel" name="label_28">
     <property name="text">
      <string>循环次数</string>
     </property>
    </widget>
   </item>
   <item row="17" column="1">
    <widget class="QLineEdit" name="numLoop"/>
   </item>
   <item row="18" column="0">
    <widget class="QLabel" name="label_21">
     <property name="text">
      <string>过渡周期</string>
     </property>
    </widget>
   </item>
   <item row="18" column="1">
    <widget class="QLineEdit" name="middleTime"/>
   </item>
   <item row="19" column="0">
    <widget class="QLabel" name="label_22">
     <property name="text">
      <string>颜色变化</string>
     </property>
    </widget>
   </item>
   <item row="19" column="1">
    <widget class="QColorPicker" name="startColor"/>
   </item>
   <item row="20" column="1">
    <widget class="QColorPicker" name="midColor"/>
   </item>
   <item row="21" column="1">
    <widget class="QColorPicker" name="endColor"/>
   </item>
   <item row="22" column="0">
    <widget class="QLabel" name="label_25">
     <property name="text">
      <string>大小变化</string>
     </property>
    </widget>
   </item>
   <item row="22" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLineEdit" name="startSize"/>
     </item>
     <item>
      <widget class="QLineEdit" name="midSize"/>
     </item>
     <item>
      <widget class="QLineEdit" name="endSize"/>
     </item>
    </layout>
   </item>
   <item row="23" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>粒子贴图</string>
     </property>
    </widget>
   </item>
   <item row="23" column="1">
    <widget class="QComboBox" name="textureId"/>
   </item>
   <item row="24" column="0">
    <widget class="QLabel" name="label_23">
     <property name="text">
      <string>旋转角度</string>
     </property>
    </widget>
   </item>
   <item row="24" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLineEdit" name="rotVec1"/>
     </item>
     <item>
      <widget class="QLineEdit" name="rotVec2"/>
     </item>
     <item>
      <widget class="QLineEdit" name="rotVec3"/>
     </item>
    </layout>
   </item>
   <item row="25" column="0">
    <widget class="QLabel" name="label_20">
     <property name="text">
      <string>旋转速度</string>
     </property>
    </widget>
   </item>
   <item row="25" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLineEdit" name="rotVel1"/>
     </item>
     <item>
      <widget class="QLineEdit" name="rotVel2"/>
     </item>
     <item>
      <widget class="QLineEdit" name="rotVel3"/>
     </item>
    </layout>
   </item>
   <item row="26" column="0" colspan="2">
    <widget class="QCheckBox" name="randRotVec">
     <property name="text">
      <string>启用随机角度</string>
     </property>
    </widget>
   </item>
   <item row="27" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>随机角度</string>
     </property>
    </widget>
   </item>
   <item row="27" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="rotVecVar11"/>
     </item>
     <item>
      <widget class="QLineEdit" name="rotVecVar21"/>
     </item>
     <item>
      <widget class="QLineEdit" name="rotVecVar31"/>
     </item>
    </layout>
   </item>
   <item row="28" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <widget class="QLineEdit" name="rotVecVar12"/>
     </item>
     <item>
      <widget class="QLineEdit" name="rotVecVar22"/>
     </item>
     <item>
      <widget class="QLineEdit" name="rotVecVar32"/>
     </item>
    </layout>
   </item>
   <item row="29" column="0" colspan="2">
    <widget class="QCheckBox" name="randRotVel">
     <property name="text">
      <string>启用随机转速</string>
     </property>
    </widget>
   </item>
   <item row="30" column="0">
    <widget class="QLabel" name="label_27">
     <property name="text">
      <string>随机转速</string>
     </property>
    </widget>
   </item>
   <item row="30" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item>
      <widget class="QLineEdit" name="rotVelVar11"/>
     </item>
     <item>
      <widget class="QLineEdit" name="rotVelVar21"/>
     </item>
     <item>
      <widget class="QLineEdit" name="rotVelVar31"/>
     </item>
    </layout>
   </item>
   <item row="31" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <widget class="QLineEdit" name="rotVelVar12"/>
     </item>
     <item>
      <widget class="QLineEdit" name="rotVelVar22"/>
     </item>
     <item>
      <widget class="QLineEdit" name="rotVelVar32"/>
     </item>
    </layout>
   </item>
   <item row="32" column="0" colspan="2">
    <widget class="QCheckBox" name="enableRandSize">
     <property name="text">
      <string>启用随机大小</string>
     </property>
    </widget>
   </item>
   <item row="33" column="0">
    <widget class="QLabel" name="label_19">
     <property name="text">
      <string>随机大小</string>
     </property>
    </widget>
   </item>
   <item row="33" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLineEdit" name="randSize1"/>
     </item>
     <item>
      <widget class="QLineEdit" name="randSize2"/>
     </item>
    </layout>
   </item>
   <item row="34" column="0" colspan="2">
    <widget class="QCheckBox" name="enableBump">
     <property name="text">
      <string>启用凹凸贴图</string>
     </property>
    </widget>
   </item>
   <item row="35" column="0">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>凹凸贴图</string>
     </property>
    </widget>
   </item>
   <item row="35" column="1">
    <widget class="QComboBox" name="bumpTexId"/>
   </item>
   <item row="36" column="0">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>凹凸强度</string>
     </property>
    </widget>
   </item>
   <item row="36" column="1">
    <widget class="QLineEdit" name="bumpAmount"/>
   </item>
   <item row="37" column="0">
    <widget class="QLabel" name="label_16">
     <property name="text">
      <string>脚本</string>
     </property>
    </widget>
   </item>
   <item row="37" column="1">
    <widget class="QPlainTextEdit" name="script"/>
   </item>
   <item row="12" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLineEdit" name="tailStartLength"/>
     </item>
     <item>
      <widget class="QLineEdit" name="tailLength"/>
     </item>
    </layout>
   </item>
   <item row="13" column="1">
    <widget class="QLineEdit" name="tailSpeed"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QColorPicker</class>
   <extends>QLineEdit</extends>
   <header location="global">QColorPicker.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>name</tabstop>
  <tabstop>squirt</tabstop>
  <tabstop>modelSpace</tabstop>
  <tabstop>xyQuad</tabstop>
  <tabstop>lockEmitter</tabstop>
  <tabstop>emitterPlane</tabstop>
  <tabstop>boneId</tabstop>
  <tabstop>speed</tabstop>
  <tabstop>speedVar</tabstop>
  <tabstop>coneAngle</tabstop>
  <tabstop>lifeSpan</tabstop>
  <tabstop>emissionRate</tabstop>
  <tabstop>row</tabstop>
  <tabstop>col</tabstop>
  <tabstop>randSize1</tabstop>
  <tabstop>randSize2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
